packages/protocol/contracts/common/Registry.sol

```diff
@@ -38,25 +38,19 @@ contract Registry is IRegistry, Ownable, Initializable {
 
   /**
    * @notice Gets address associated with the given identifier.
-   * @param identifier Identifier of contract whose address we want to look up.
+   * @param identifier Identifier hash of contract whose address we want to look up.
```

Comment:
Since these are view calls should we continue to allow users to submit the string?

Reply:
I agree. more precisely, i think we can keep the old interface for everything. And thus have "retrocompatibility" :D

Reply:
While in this case having the string functions might be useful, I don't know that the reason for keeping them should be backwards compatibility. We don't need to remain backwards compatible with Alfajores, right? If anything, if we do find APIs that could be improved, we should completely replace them so that by mainnet we have as clean an interface as possible.

Reply:
Sure, but what i propose is to do something like:

1. Make change, leave old api, mark it as deprectaded
2. Deploy
3. On a later PR, after all apps have already migrated to new contractkit/etc code that call new methods, remove deprecated methods